lib/components/label/label.less

```diff
@@ -96,6 +96,14 @@
     padding: 0 var(--su2); // Helps the label visually line up with inputs
 }
 
+// TODO: is this the right place for this?
+.s-required-symbol {
+    color: var(--red-400);
+    font-size: 125%;
+    font-weight: normal;
+    text-decoration: none !important;
+}
```

Comment:
I noticed a height difference in labels with and without the required symbol due to the font-size.

<details>
<summary>Gif example</summary>

![20240429-041428](https://github.com/StackExchange/Stacks/assets/647177/cdb43a8d-fd19-4336-adcf-99781ff9fc2c)

</details>

I think we can prevent this layout shift while retaining the larger font size by setting line-height to 0 on `.s-required-symbol`.

```suggestion
.s-required-symbol {
    color: var(--red-400);
    font-size: 125%;
    font-weight: normal;
    line-height: 0;
    text-decoration: none !important;
}
```

Reply:
Nice catch @dancormier. Thank you